pkg/k8s/pods/pod.go

```diff
@@ -212,11 +213,58 @@ func Exists(podName, namespace string, c kubernetes.Interface) bool {
 	return pod.GetObjectMeta().GetDeletionTimestamp() == nil
 }
 
-//ContainerLogs returns the logs of a given container in a given pod
-func ContainerLogs(container string, pod *apiv1.Pod, namespace string, c kubernetes.Interface) (string, error) {
+//GetDevPodUserID returns the user id running the dev pod
+func GetDevPodUserID(ctx context.Context, dev *model.Dev, c *kubernetes.Clientset) int64 {
```

Comment:
could we do a `kubectl exec echo $USER` instead?

Reply:
no, at that time, the pod might not be running